event-based-simulators/main/scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# Generic scripts information
+This scripts folder contains:
+- The **ExportOrImportProfileData.py** which is used to export measurements or alarms data from devices
+- The **Environment.py** which is used to setup environment parameters
+
+## Setup environment parameters:
+Input base url, tenant ID, username, password in Environment.py file
+
+## Install cumulocity-python-api package
+Follow the instructions in: https://github.com/SoftwareAG/cumulocity-python-api
+
+### Installation from PyPI
+```shell
+pip install c8y_api
+```
+
+## Run the script
```

Comment:
- if I run `python ExportOrImportProfileData.py -h`, it shows the options, but then it fails with `NameError: name 'Credentials' is not defined`

Reply:
I mistakenly set up the name of the Environment file to Credentials.